include/API/errors.h

```diff
@@ -46,6 +46,7 @@ enum class ErrorCategory {
   QSSLinkSignatureNotFound,
   QSSLinkArgumentNotFoundWarning,
   QSSLinkInvalidPatchTypeError,
+  QSSControlSystemResourcesExceeded,
```

Comment:
```suggestion
  QSSTargetSystemResourcesExceeded,
```
Perhaps this is better for the qe-compiler?

Reply:
I'm not sure those mean similar enough things to make sense in translation. I would think that `QSSTargetSystemResourcesExceeded` error category would apply to a circuit that included too many qubits for example.